<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="msgsendButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filesendButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件传输</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgtextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="imagesendButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送图片</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="msg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>531</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="muliLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>421</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="kachaButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cameraButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开启摄像头</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
